src/gui/properties/propertieswidget.cpp

```diff
@@ -193,24 +193,31 @@ void PropertiesWidget::showPiecesDownloaded(bool show)
 
 void PropertiesWidget::setVisibility(bool visible)
 {
+    static int handleWidth = 0; // not 5 everywhere
     if (!visible && (m_state == VISIBLE)) {
         auto *hSplitter = static_cast<QSplitter *>(parentWidget());
+        const int tabBarHeight = m_tabBar->geometry().height(); // take the height before we hide the properties else it will resize to take whole widget
         m_ui->stackedProperties->setVisible(false);
         m_slideSizes = hSplitter->sizes();
         hSplitter->handle(1)->setVisible(false);
         hSplitter->handle(1)->setDisabled(true);
-        QList<int> sizes = QList<int>() << hSplitter->geometry().height() - 30 << 30;
+        handleWidth = hSplitter->handleWidth();
+        hSplitter->setHandleWidth(0);
+        QList<int> sizes = QList<int>() << hSplitter->geometry().height() - tabBarHeight << tabBarHeight;
         hSplitter->setSizes(sizes);
+        setFixedHeight(tabBarHeight);
```

Comment:
If possible the size police of the relevant widgets should be tweaked instead of fixing the height. I would like to experiment that idea myself later when I have more time.

Reply:
I tried with the tweaking size policy of PropertiesWidget, but it didn't work out. setFixedHeight function is the fix of #11622. I can remove this and submit as separate PR.

Reply:
I also tried with wrapping m_tabbar in a widget and changing the size policy of widget. But it didn't work as well. IDK maybe I am missing something here.